src/puppetlabs/trapperkeeper/bootstrap.clj

```diff
@@ -132,26 +136,102 @@
             (str "Unable to find bootstrap.cfg file via --bootstrap-config "
                  "command line argument, current working directory, or on classpath")))))
 
-(schema/defn chain-files :- [schema/Str]
-  "Takes a list of files, reads all their lines in, and returns a flattened seq
-   of all their lines put together.
+(schema/defn indexed
+  "Returns seq of [index, item] pairs
+  [:a :b :c] -> ([0 :a] [1 :b] [2 :c])"
+  [coll]
+  (map vector (iterate inc 0) coll))
```

Comment:
I think since zipmap returns a map we'll have ordering issues, but it looks like `(map vector (range) coll)` works